centos-quarkus-maven/Dockerfile

```diff
@@ -5,34 +5,37 @@ FROM centos:7
 
 ARG GRAAL_VERSION=1.0.0-rc14
 ARG MAVEN_VERSION=3.6.0
-
-ENV GRAAL_CE_URL=https://github.com/oracle/graal/releases/download/vm-${GRAAL_VERSION}/graalvm-ce-${GRAAL_VERSION}-linux-amd64.tar.gz
+ARG USER_HOME_DIR="/root"
+ARG GRAAL_CE_URL=https://github.com/oracle/graal/releases/download/vm-${GRAAL_VERSION}/graalvm-ce-${GRAAL_VERSION}-linux-amd64.tar.gz
 ARG MAVEN_URL=https://archive.apache.org/dist/maven/maven-3/$MAVEN_VERSION/binaries/apache-maven-$MAVEN_VERSION-bin.tar.gz
-ARG INSTALL_PKGS="buildah findutils podman bzip2-devel gcc-c++ libcurl-devel openssl-devel tar unzip bc which lsof gzip"
+ARG INSTALL_PKGS="buildah findutils podman tar gzip gcc glibc-devel zlib-devel curl"
```

Comment:
Any reason to remove the other libs? Typically openssl is required

Reply:
I guess I made mistake tried to make the images shrink in size, but its failing to build properly because of missing libraries. will revert it back